Sharpnado.CollectionView.Droid/Renderers/CollectionViewRenderer.RecycleViewAdapter.cs

```diff
@@ -321,16 +335,29 @@ public void OnItemMoving(int from, int to)
             protected override void Dispose(bool disposing)
             {
                 _isDisposed = true;
+                
                 _viewHolderQueue?.Clear();
 
                 if (_notifyCollectionChanged != null)
                 {
                     _notifyCollectionChanged.CollectionChanged -= OnCollectionChanged;
                 }
 
+
+
+                
+                _dataSource.Clear();
+                _dataSourceItemViewType.Clear();
+                _formsViews.Clear();
+                foreach (var viewHolder in createdViewHolders)
+                {
+                    viewHolder.Dispose();
+                }
+                createdViewHolders.Clear();
```

Comment:
```suggestion
                }
                
                createdViewHolders.Clear();
```
add blank line after }